pubspec.yaml

```diff
@@ -6,10 +6,11 @@ homepage: https://github.com/tappeddev/injector
 author: Tapped <contact@tapped.dev>
 
 environment:
-  sdk: '>=1.20.1 <3.0.0'
+  sdk: '>=2.7.2 <3.0.0'
```

Comment:
Is `2.7.2` the minimum Version possible? If not I would try to lower it as much as possible.

Reply:
True 🙈

Reply:
2.6.0 works as well ✅